qa/tasks/cephfs/test_volumes.py

```diff
@@ -35,18 +35,22 @@ def _fs_cmd(self, *args):
     def _raw_cmd(self, *args):
         return self.get_ceph_cmd_stdout(args)
 
-    def __check_clone_state(self, state, clone, clone_group=None, timo=120):
+    def __check_clone_state(self, states, clone, clone_group=None, timo=120,
+                            sleep=1):
+        if isinstance(states, str):
+            states = (states, )
+
         check = 0
         args = ["clone", "status", self.volname, clone]
         if clone_group:
             args.append(clone_group)
         args = tuple(args)
         while check < timo:
             result = json.loads(self._fs_cmd(*args))
-            if result["status"]["state"] == state:
+            if result["status"]["state"] in states:
```

Comment:
Done.

Reply:
> why is it really required since if a long enough timeout is used before checking the state (or better, using safe_while), then checking multiple states isn't really necessary.

Reaching `timeout` would halt the tests. That's not what is desired here. The idea for checking for multiple states is accepted clone might reaching completion while or before the command for clone cancellation is launched.

Reply:
Marking this as resolved.